vizro-core/changelog.d/20230912_165919_huong_li_nguyen_enable_dashboard_title.md

```diff
@@ -0,0 +1,41 @@
+<!--
+A new scriv changelog fragment.
+
+Uncomment the section that is right (remove the HTML comment wrapper).
+-->
+
+<!--
+### Removed
+
+- A bullet item for the Removed category.
+
+-->
+
+### Added
+
+- Enabled `title` in `Dashboard` ([#31](https://github.com/mckinsey/vizro/pull/31))
```

Comment:
```suggestion
- Enable `title` argument in `Dashboard` model, which allows a title to be added to every page above the navigation panel ([#31](https://github.com/mckinsey/vizro/pull/31))
```

Reply:
Rephrased to `Enable `title` argument in `Dashboard` model, which allows a title to be added on every page on top left-side` because the navigation panel does not always exist on the page.